tq42/exception_handling.py

```diff
@@ -26,9 +26,9 @@ def wrapped(*args, **kwargs):
             if status_code == StatusCode.UNAUTHENTICATED:
                 raise exceptions.UnauthenticatedError() from None
 
-            if status_code == StatusCode.INVALID_ARGUMENT:
+            if status_code == StatusCode.INVALID_ARGUMENT or status_code == StatusCode.NOT_FOUND:
```

Comment:
Please use this syntax as it is more extensible:
`if status_code in [StatusCode.INVALID_ARGUMENT, StatusCode.NOT_FOUND]`